src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -1646,7 +1646,10 @@ async function webviewPreloads(ctx: PreloadContext) {
 			this.outputItem = Object.freeze(<rendererApi.OutputItem>{
 				id,
 				mime,
-				metadata,
+
+				metadata: (): NotebookCellMetadata => {
```

Comment:
`metadata` should be the metadata object itself instead of a function that returns the object

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/vscode-notebook-renderer/index.d.ts#L49

You can use a getter to implement this though